docs/src/api/request_collection/chain_source.md

```diff
@@ -135,4 +136,13 @@ Prompt the user for input to use as the rendered value.
 | Field     | Type       | Description                                                                                                                                   | Default  |
 | --------- | ---------- | --------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
 | `message` | `Template` | Descriptive prompt for the user                                                                                                               | Chain ID |
-| `default` | `Template` | Value to pre-populated the prompt textbox. **Note**: Dur to a library limitation, not supported on chains with `sensitive: true` _in the CLI_ | `null`   |
+| `default` | `Template` | Value to pre-populated the prompt textbox. **Note**: Due to a library limitation, not supported on chains with `sensitive: true` _in the CLI_ | `null`   |
+
+### Select
+
+Prompt the user to select a defined value from a list.
+
+| Field     | Type          | Description                                  | Default  |
+| --------- | ------------- | -------------------------------------------- | -------- |
+| `message` | `Template`    | Descriptive prompt for the user              | Chain ID |
+| `options` | `Template[]`  | The list of options to present to the user   | Required |
```

Comment:
```suggestion
| `options` | `Template[]`  | List of options to present to the user   | Required |
```

(sorry for the nitpick, I just like to be as brief as possible in docs)

Reply:
Nah, not nit-picking! Language that sounds consistent is important 🤷